src/AppLayout.tsx

```diff
@@ -31,7 +32,9 @@ function AppLayout() {
             </Box>
 
             <Box sx={{ ml: `${navigationWidth}px` }}>
-                <Outlet />
+                <ErrorBoundryWrapper>
+                    <Outlet />
+                </ErrorBoundryWrapper>
```

Comment:
This just handled a weird edge case while figuring out the new routing. However, it is probably safer to include this so the entire page doesn't blow up.